tools/nni_annotation/code_generator.py

```diff
@@ -316,16 +320,19 @@ def _visit_string(self, node):
             return node  # not an annotation, ignore it
 
         if string.startswith('@nni.get_next_parameter'):
-            deprecated_message = "'@nni.get_next_parameter' is deprecated in annotation due to inconvenience. Please remove this line in the trial code."
-            print_warning(deprecated_message)
+            call_node = parse_annotation(string[1:]).value
```

Comment:
please add comments here: it is used in enas mode as it needs to retrieve the next subgraph for training.

Reply:
resolve